crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -320,7 +371,7 @@ fn is_approximately_equal(executed_value: U256, expected_value: U256) -> bool {
 
 async fn execute_test(
     web3: Web3,
-    fee_policy: FeePolicyKind,
+    autopilot_config: Vec<String>,
```

Comment:
nit: don't really feel confident to use strings here. Would rather see a protocol fees config struct.

Reply:
Sure! I am changing to accept anything which implements `ToString`. The reason of it is that we can keep the current `FeePolicyKind` enum without needing to convert it to string, and at the same time it adds customization to what you want to pass. 

The idea of doing it like that is to avoid creating a struct containing `FeePolicyKind` and a new field which is going to be used just is one test, and at the same time it opens to possibility of adding more tests with different configurations without altering the `execute_test()` signature.